README.md

```diff
@@ -4,52 +4,68 @@ This is a **group exercise**, so you should be working in pairs of two students.
 
 The Goal is to **practise writing readable, maintainable and reliable code collaboratively.**
 
-## Group Exercise
 
-1. One student of your group forks the code from [https://github.com/advanced-geoscripting-2021/random_walker.git](https://github.com/advanced-geoscripting-2021/random_walker.git)
+## Write an extended random walk program 
 
-2. This student invites the other student as a collaborator to the forked repository. Now you can both work on the code.
+In this repo you find a basic implementation of a [random walk simulation](https://en.wikipedia.org/wiki/Random_walk) in 2-dimensional space taken from [this blogpost](https://www.geeksforgeeks.org/random-walk-implementation-python/). Running the code yields an image which shows the path of the random walk. 
 
-3. Adapt the code to fulfil the requirements (see below).
+![random_walk](rand_walk_5_100.png)
```

Comment:
Good idea to mark starting and endpoint! Maybe you could describe which symbol is the start/end, but looks good and is some useful additional info